resources/files/sacct/slurm_rest_api.rb

```diff
@@ -0,0 +1,148 @@
+require 'json'
+return if node['cluster']['node_type'] != 'HeadNode'
+
+slurm_etc = '/opt/slurm/etc'
+state_save_location = '/var/spool/slurm.state'
+key_location = state_save_location + '/jwt_hs256.key'
+certs_location = '/etc/ssl/certs'
+key_and_crt_name = 'nginx-selfsigned'
+id = 2005
+
+platform = node['platform']
+if platform == 'amazon'
+  platform = 'amzn2'
+end
+
+# Configure Slurm for JWT authentication and enable slurmrestd
+ruby_block 'Create JWT key file' do
+  block do
+    shell_out!("dd if=/dev/random of=#{key_location} bs=32 count=1")
+  end
+end
+
+# TODO: Not idempotent if user is in process 
+group 'slurmrestd' do
+    comment 'slurmrestd group'
+    gid id
+    system true
+end
+
+user 'slurmrestd' do
+  comment 'slurmrestd user'
+  uid id
+  gid id
+  home '/home/slurm'
```

Comment:
does it need to have a shared home? can we even disable home creation for slurmrestd user?

Reply:
I carried this over from [this PR](https://github.com/aws/aws-parallelcluster-cookbook/pull/1421/files), but after some testing it looks like this isn't necessary and I can just remove this line.